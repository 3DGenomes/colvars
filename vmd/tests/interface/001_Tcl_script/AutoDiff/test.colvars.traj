# step         rmsd                  fa_rmsd                rmsd-all               E_harmonic1           x0_rmsd               W_harmonic1          
           0    7.02166693715340e-17 -7.02166693715340e-13   0.00000000000000e+00   2.46519032881566e-29  0.00000000000000e+00 -7.02166693715340e-15
           1    3.16374866030856e-03  6.83625133969145e+01   3.06562197951409e-03   2.33671661897165e-01  1.00000000000000e-02  6.83625133969137e-01
           2    6.43042836588687e-03  1.35695716341131e+02   6.20782631005232e-03   9.20666371666639e-01  2.00000000000000e-02  2.04058229738045e+00
           3    9.79623008237727e-03  2.02037699176227e+02   9.43628625319814e-03   2.04096159442119e+00  3.00000000000000e-02  4.06095928914272e+00
           4    1.32402365614076e-02  2.67597634385924e+02   1.27755653235612e-02   3.58042469644714e+00  4.00000000000000e-02  6.73693563300197e+00
           5    1.67526858867062e-02  3.32473141132938e+02   1.62397875984789e-02   5.52691947874013e+00  5.00000000000000e-02  1.00616670443313e+01
           6    2.05399880743223e-02  3.94600119256777e+02   1.98096952490138e-02   7.78546270587313e+00  6.00000000000000e-02  1.40076682368991e+01
           7    2.46472075956546e-02  4.53527924043454e+02   2.34704400098299e-02   1.02843788943583e+01  7.00000000000000e-02  1.85429474773337e+01
           8    2.88885276374936e-02  5.11114723625064e+02   2.71557732913004e-02   1.30619130353163e+01  8.00000000000000e-02  2.36540947135843e+01
           9    3.34520835953681e-02  5.65479164046319e+02   3.08846350501963e-02   1.59883342485262e+01  9.00000000000000e-02  2.93088863540475e+01
          10    3.82640246532821e-02  6.17359753467179e+02   3.46452320983256e-02   1.90566532600528e+01  1.00000000000000e-01  3.54824838887193e+01
