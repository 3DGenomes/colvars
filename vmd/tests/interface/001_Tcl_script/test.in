# adapted from NAMD's 3rd reg test

colvarsTrajFrequency    1
colvarsRestartFrequency 0


colvar {
    name rmsd

#    outputSystemForce on
    outputAppliedForce on
    width 0.01

    lowerBoundary 0.0
    upperBoundary 0.1

    hardLowerBoundary yes
    expandBoundaries yes

    lowerWallConstant 10.0
    upperWallConstant 10.0

    rmsd {
        atoms {
            psfSegID BH
            atomNameResidueRange CA 1-10
        }
        refpositionsfile ../Common/da.pdb
    }
} 

harmonic {
    colvars        rmsd
    centers        0.0
    targetCenters  0.2
    targetNumSteps 20
    outputEnergy   yes
    outputAccumulatedWork  yes
    outputCenters  yes
}
