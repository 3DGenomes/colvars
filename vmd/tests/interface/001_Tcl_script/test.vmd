# -*- tcl -*-
mol new ../Common/da.psf type psf
# this is the file from NAMD's 3rd reg test: it is XYZ format to gain more precision
mol addfile ../Common/da.test-traj.xyz type xyz waitfor all

cv molid top
cv configfile test.in

set out [open "test.colvars.traj" "w"]
puts -nonewline ${out} [cv printframelabels]
for { set frame 0 } { ${frame} < [molinfo top get numframes] } { incr frame } {
    cv frame ${frame}
    cv update
    puts -nonewline ${out} [cv printframe]
}

quit
