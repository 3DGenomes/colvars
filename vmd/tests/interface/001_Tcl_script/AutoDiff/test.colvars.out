colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  http://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 0
colvars: # colvarsTrajAppend = off [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = on [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rmsd"
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # scalable = on [default]
colvars:     Scalable calculation is not available for group "atoms" with the current configuration.
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "atoms" defined, 10 atoms initialized: total mass = 120.11, total charge = 0.53.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da.pdb"
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating it as a variable: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:     This is a standard minimum RMSD, derivatives of the optimal rotation will not be computed as they cancel out in the gradients.
colvars:   All components initialized.
colvars:   # width = 0.01
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 10
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 0.1
colvars:   # upperWallConstant = 10
colvars:   # upperWall = 0.1 [default]
colvars:   # hardLowerBoundary = on
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = on
colvars:   # timeStepFactor = 1 [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = off [default]
colvars:   # outputAppliedForce = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rmsd }
colvars:   # outputEnergy = on
colvars:   # forceConstant = 1 [default]
colvars:   # centers = { 0 }
colvars:   # targetCenters = { 0.2 }
colvars:   # targetForceConstant = 0 [default]
colvars:   # targetNumSteps = 20
colvars:   # targetNumStages = 0 [default]
colvars:   # targetForceExponent = 1 [default]
colvars:   # outputCenters = on
colvars:   # outputAccumulatedWork = on
colvars:   The force constant for colvar "rmsd" will be rescaled to 10000 according to the specified width.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration:
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1 [default]
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # colvarsTrajAppend = off [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = on [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rmsd-all"
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # scalable = on [default]
colvars:     Scalable calculation is not available for group "atoms" with the current configuration.
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "atoms" defined, 51 atoms initialized: total mass = 674.397, total charge = -7.64.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da.pdb"
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating it as a variable: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:     This is a standard minimum RMSD, derivatives of the optimal rotation will not be computed as they cancel out in the gradients.
colvars:   All components initialized.
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # upperWallConstant = 0 [default]
colvars:   # expandBoundaries = off [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
